server/sonar-web/src/main/js/apps/projectActivity/components/GraphHistory.js

```diff
@@ -64,6 +64,8 @@ export default class GraphHistory extends React.PureComponent {
   formatValue = (tick /*: string | number */) =>
     formatMeasure(tick, getShortType(this.props.metricsType));
 
+  formatTooltipValue = (tick /*: string | number */) => formatMeasure(tick, this.props.metricsType);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)